src/libs/actions/Report.js

```diff
@@ -1595,7 +1595,13 @@ function removeEmojiReaction(reportID, reportActionID, emoji) {
  * @param {Object} existingReactions
  * @param {Number} [paramSkinTone]
  */
-function toggleEmojiReaction(reportID, reportAction, emoji, existingReactions, paramSkinTone = preferredSkinTone) {
+function toggleEmojiReaction(reportID, reportActionID, emoji, existingReactions, paramSkinTone = preferredSkinTone) {
+    const reportAction = ReportActionsUtils.getReportAction(reportID, reportActionID);
```

Comment:
Apparently this doesn't work for the first message in a threads, but we have [another issue](https://github.com/Expensify/App/issues/22235) to fix it so it's ok for now.